astropy/io/fits/hdu/table.py

```diff
@@ -790,6 +785,30 @@ class BinTableHDU(_TableBaseHDU):
     _extension = 'BINTABLE'
     _ext_comment = 'binary table extension'
 
+    def __init__(self, data=None, header=None, name=None, uint=False):
+        """
+        Parameters
+        ----------
+        header : `Header`
+            header to be used
+        data : array, `FITS_rec` or `astropy.table.Table`
+            data to be used
+        name : str
+            name to be populated in ``EXTNAME`` keyword
+        uint : bool, optional
```

Comment:
It's part of the Numpy docstring convention, but `optional` is optional I think :grin: . Again no preference from my side, but it's used in some parts of astropy and in this file in particular, and I agree it would be better to have something consistent ...